### ISSUE TITLE
[FEATURE REQUEST] Live/Development/Watch mode

### ISSUE THREAD

**Issue by dalisoft** [pgrange/bash_unit#112]
Like when calling `./bash_unit tests/**/*.test.sh --watch`, it should always track and whenever `test` changes (only changed test) should be re-triggered

**Comment by pgrange** [comment]
Hello @dalisoft,

It seems to me like a real cool feature. I'm not planning to address it for now though but feel free to submit a pull request if you have something in mind. In the meantime, I'll keep it open for when I've some time to hack on it.

Cheers,

**Comment by dalisoft** [comment]
@pgrange I don't how to implement it properly cross-platform (Unix, Linux). But for now `nodemon` could be used

**Comment by pgrange** [comment]
Me too, the multi-platform part is tricky and I want to avoid introducing external dependencies to bash_unit, as much as possible

**Comment by dalisoft** [comment]
Yea, i understand, so only currently have no idea how do it properly

**Comment by pgrange** [comment]
Fair enough. Let's keep it open. I think it's a nice to have, maybe I can find some time in the future to dig into it 👍
